operate/importer/src/main/java/io/camunda/operate/zeebeimport/ImportPositionHolder.java

```diff
@@ -124,15 +129,28 @@ public void recordLatestLoadedPosition(final ImportPositionEntity lastProcessedP
           ImportPositionEntity importPosition = inflightImportPositions.get(key);
           if (importPosition == null) {
             importPosition = lastProcessedPosition;
+            LOGGER.error("SET import position {}; NOT EXIST BEFORE", importPosition);
+            importPositionCompletedGauges.put(
+                key,
+                metrics.registerGauge(
+                    Metrics.GAUGE_NAME_IMPORT_POSITION_COMPLETED,
+                    importPosition,
+                    (pos) -> pos.getCompleted() ? 1.0 : 0.0,
+                    Metrics.TAG_KEY_PARTITION,
+                    Integer.toString(partition),
+                    Metrics.TAG_KEY_IMPORT_POS_ALIAS,
+                    aliasName));
           } else {
             importPosition
                 .setPosition(lastProcessedPosition.getPosition())
                 .setSequence(lastProcessedPosition.getSequence())
                 .setIndexName(lastProcessedPosition.getIndexName())
                 .setCompleted(lastProcessedPosition.getCompleted());
+            LOGGER.error("SET import position {}; HAS EXIST BEFORE", importPosition);
```

Comment:
@EuroLew We had some debugging logs added because of the failed tests 😅 They are now on main. Can you remove them again please?